build.sbt

```diff
@@ -61,6 +63,7 @@ lazy val root = project
     http4s,
     akkaHttp,
     uzhttp,
+    play,
```

Comment:
Can you also add this to the config file in `.circleci`?

Reply:
done!